PROTOCOL.md

```diff
@@ -260,6 +260,12 @@ etc). `NAME` is the nickname of the Pokemon performing the action.
 > `switch` means it was intentional, while `drag` means it was unintentional
 > (forced by Whirlwind, Roar, etc).
 
+`|swap|POKEMON|POSITION|CAUSE`
+
+> Moves already active `POKEMON` to active field `POSITION` where `0` is the
+> leftmost position and each position to the right increments by 1. `CAUSE`
+> is optional and only used for debug.
```

Comment:
I'd use the same wording as the action requests you have, saying that it counts up from 0 starting from the leftmost position.

Also, is `CAUSE` stuff like `[from] blah|[of] bleh` and what not? If so that part was covered all the way at the top of the battle protocol section.